dmscripts/bulk_upload_documents.py

```diff
@@ -0,0 +1,39 @@
+import os
+import re
+import sys
+sys.path.insert(0, '.')
+from dmutils.documents import get_countersigned_agreement_document_path
```

Comment:
(And as this script is no longer only for countersigned agreements we should probably have a more generic sounding method to use anyway)